flake/dev/flake.lock

```diff
@@ -0,0 +1,110 @@
+{
+  "nodes": {
+    "dev-nixpkgs": {
+      "locked": {
+        "lastModified": 1746300365,
+        "narHash": "sha256-thYTdWqCRipwPRxWiTiH1vusLuAy0okjOyzRx4hLWh4=",
+        "owner": "NixOS",
+        "repo": "nixpkgs",
+        "rev": "f21e4546e3ede7ae34d12a84602a22246b31f7e0",
```

Comment:
Same, fix with:

```sh
nix flake lock ./flake/dev --override-input nixpkgs 'github:NixOS/nixpkgs/46e634be05ce9dc6d4db8e664515ba10b78151ae'
```

(Note, this is additionally out-of-sync with the root flake.lock, so as-is you'd download two nixpkgs revisions (`f21e4546` & `7a2622e2`))